docs/community.html

```diff
@@ -0,0 +1,231 @@
+  </ul>
+
+
+    </nav>
+  </header>
+  
+
+  <main class="ov-main">
+   <section class="ov-masthead--community ov-masthead">
+      <h1 class="ov-heading ov-heading--xl">Get Involved</h1>
+      <h2 class="ov-heading ov-heading--lg">oVirt is a community driven virtualization project and people just like you are making it happen.</h2>
+    </section>
+
+<div class="ov-site-section">
+   <section class="ov-community-subsection">
+      <h2 class="ov-heading ov-heading--lg">Community</h2>
+        <p class="ov-content-styles"> oVirt is a community project, and we welcome contributions from everyone! If you’d like to write code, report bugs, contribute designs, or enhance the documentation, we would love your help!
+        </p>    
+        <p class="ov-content-styles">
+        There are a few ways to engage with the oVirt Community: 
+        </p>
+        <ul class="ov-content-styles">
+          <li>
+            <a href="https://ovirt.org/events/">Events</a>
+          </li>
+          <li>
+            <a href="#">Forums/ Mailing lists</a>
+          </li>
+          <li>
+            <a href="https://www.facebook.com/groups/ovirt.openvirtualization/">oVirt Facebook Group</a>
+          </li>
+          <li>
+            <a href="https://twitter.com/ovirt">Twitter</a>
+          </li>
+          <li>
+            IRC – #ovirt on irc.oftc.net
+          </li>
+        </ul>
+        <p class="ov-content-styles">
+          Before getting started, please read our <a href="https://ovirt.org/community/about/community-guidelines.html">community etiquette guidelines</a>.
+        </p>
+    </section>
+
+    <section class="ov-community-subsection">
+      <h2 class="ov-heading ov-heading--lg">Developers</h2>
+      <div>
+        <p class="ov-content-styles ">
+          If you’d like to contribute code to oVirt, visit the <a href="https://ovirt.org/develop/">Developer section</a> of the site for guidelines. All of our projects use git. Most are hosted at <a href="https://gerrit.ovirt.org/#/dashboard/self">gerrit.ovirt.org</a> , and the rest are hosted at <a href="https://github.com/oVirt/"> github.com/oVirt</a>. 
+        </p>  
+      </div>  
+    </section>
+
+    <section class="ov-community-subsection">
+      <h2 class="ov-heading ov-heading--lg">Translate</h2>
+      <div>
+        <p class="ov-content-styles">
+          oVirt’s user interface and documentation are available in multiple languages for users around the globe. Examples include Italian, Portuguese, Chinese, Japanese, Korean, and more. To contribute translations, sign up for <a href="https://translate.zanata.org/?dswid=8280">Zanata</a> and access our <a href="https://translate.zanata.org/project/view/ovirt?dswid=1139">project</a>. 
+        </p>    
+      </div>  
+    </section>
+
+    <section class="ov-community-subsection">
+      <h2 class="ov-heading ov-heading--lg">Documentation</h2>
+      <div>
+        <p class="ov-content-styles">
```

Comment:
I'd apply ov-content-styles to the parent wrapper and then define all the html elements styles under that. 
so something like 
```
.ov-content-styles {
p {
margin-bottom: var(--ov-0-margin--sm);
}
ul {
list-style: disc;
}
}
```
This makes it so just need one class for all the styles and you don't need to add it to every element.